bucket/fio.json

```diff
@@ -1,20 +1,32 @@
 {
-    "version": "3.20",
+    "version": "3.27",
     "description": "Flexible I/O Tester",
     "homepage": "https://bsdio.com/fio/",
```

Comment:
Is the homepage for Fio correct @Valinor ?

This is Rebecca Cran's fio package homepage right?

If this isn't, we could fix it in a future PR

Reply:
I don't know, there is no official webpage, only the git link : https://git.kernel.dk/cgit/fio/
I think the Rebecca page is best.